tests/other/stress_test.js

```diff
@@ -12,8 +12,7 @@ See LICENSE file in root for details.
 
 *******************************************************************************/
 
-import axios from 'axios';
-
+import { post } from '../../lib/fetch.js';
```

Comment:
In this file there is still `axios.get` used instead of a new logic for the `get`.

Reply:
You're right, fixed ✅